src/main/java/roomescape/domain/member/service/MemberService.java

```diff
@@ -50,7 +50,7 @@ public Member findByToken(String token) {
 
     @Transactional
     public Member save(MemberRequest memberRequest) {
-        Long id = memberRepository.save(new Member(null, memberRequest.getName(), memberRequest.getEmail(), memberRequest.getPassword()));
+        Long id = memberRepository.save(new Member(null, memberRequest.getName(), memberRequest.getEmail(), memberRequest.getPassword(), "USER"));
```

Comment:
USER, ADMIN 과 같은 값은 enum으로 빼는게 유지보수에 좋아 보입니다.

```suggestion
        Long id = memberRepository.save(new Member(null, memberRequest.getName(), memberRequest.getEmail(), memberRequest.getPassword(), MemberRole.USER));
```

혹은 static factory method를 사용할 수도 있겠네요.

```
Member.createUser(...);
Member.createAdmin(...);
```